utils/chsmack.c

```diff
@@ -125,12 +130,15 @@ static void modify_transmute(const char *path)
 	int rc;
 	switch (transmute_flag) {
 	case positive:
-		rc = follow_flag ?  stat(path, &st) : lstat(path, &st);
+		rc = follow_flag ? stat(path, &st) : lstat(path, &st);
 		if (rc < 0)
 			perror(path);
 		else if (!S_ISDIR(st.st_mode)) {
-			fprintf(stderr, "%s: transmute: not a directory\n",
-				path);
+			if (!recursive_flag) {
```

Comment:
reason is to keep it simple when "chsmack --transmute --recursive --access LABEL file1 file2 dir1 dir2"